content/en/opentelemetry/setup/ddot_collector/install/kubernetes_gateway.md

```diff
@@ -0,0 +1,514 @@
+- link: "/opentelemetry/setup/ddot_collector/custom_components"
+  tag: "Documentation"
+  text: "Use Custom OpenTelemetry Components with Datadog Agent"
+- link: "https://opentelemetry.io/docs/collector/deployment/gateway/"
+  tag: "OpenTelemetry"
+  text: "Collector Deployment: Gateway"
+- link: "https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/loadbalancingexporter"
+  tag: "OpenTelemetry"
+  text: "Load Balancing Exporter"
+- link: "https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor"
+  tag: "OpenTelemetry"
+  text: "Tail Sampling Processor"
+---
+
+{{< callout header="false" btn_hidden="true">}}
+  Support for installing the DDOT Collector as a gateway on Kubernetes is in Preview.
+{{< /callout >}}
+
+<div class="alert alert-info">
+This guide assumes you are familiar with deploying the DDOT Collector as a daemonset. For more information, see <a href="/opentelemetry/setup/ddot_collector/install/kubernetes_daemonset">Install the DDOT Collector as a Daemonset on Kubernetes</a>.
+</div>
+
+## Overview
+
+The OpenTelemetry Collector can be deployed in multiple ways. The *daemonset* pattern is a common deployment where a Collector instance runs on every Kubernetes node alongside the core Datadog Agent.
+
+{{< img src="opentelemetry/embedded_collector/ddot_daemonset.png" alt="Architecture diagram of the OpenTelemetry Collector daemonset pattern. A Kubernetes cluster contains three nodes. On each node, an application instrumented with OpenTelemetry sends OTLP data to a local agent daemonset. The agent daemonset then forwards this data directly to the Datadog backend." style="width:100%;" >}}
+
+The [gateway][6] pattern provides an additional deployment option that uses a centralized, standalone Collector service. This gateway layer can perform actions such as tail-based sampling, aggregation, filtering, and routing before exporting the data to one or more backends such as Datadog. It acts as a central point for managing and enforcing observability policies.
+
+{{< img src="opentelemetry/embedded_collector/ddot_gateway.png" alt="Architecture diagram of the OpenTelemetry Collector gateway pattern. Applications send OTLP data to local agent daemonsets running on each node. The daemonsets forward this data to a central load balancer, which distributes it to a separate deployment of gateway Collector pods. These gateway pods then process and send the telemetry data to Datadog." style="width:100%;" >}}
+
+When you enable the gateway:
+1.  A Kubernetes Deployment (`<RELEASE_NAME>-datadog-otel-agent-gateway-deployment`) manages the standalone **gateway Collector pods**.
+2.  A Kubernetes Service (`<RELEASE_NAME>-datadog-otel-agent-gateway`) exposes the gateway pods and provides load balancing.
+3.  The existing **daemonset Collector pods** are configured by default to send their telemetry data to the gateway service instead of directly to Datadog.
```

Comment:
```suggestion
3.  The existing **DaemonSet Collector pods** are configured by default to send their telemetry data to the gateway service instead of directly to Datadog.
```